lib/rubyhub/operations/pull_request/create.rb

```diff
@@ -17,6 +21,36 @@ def call(options)
 
           def data
             @data ||= Rubyhub::Configuration.instance.options.dig(:template, template.to_sym)
+            add_description_to_data if @data
+            @data
+          end
+
+          def check_config_existence
+            raise ConfigFileDoesNotExistError unless Rubyhub::Configuration.exists?
+          end
+
+          def check_template_existence
+            raise IncorrectTemplateError if !template || !data || data.empty?
+          end
+
+          def add_description_to_data
+            @data[:description_main_body] = Rubyhub::Configuration.instance.main_body
+          end
+
+          def initialize_description_file
+            FileUtils.cp full_template_path, target_path, preserve: true, verbose: false
+          end
```

Comment:
its not good approach to initialize it here. Actually, many methods here copied from another class, isn't it? (Rubyhub::Operations::Configuration::Setup) 

I think description initialize should be right with config initialization. Please, move it all there.